content/v3/pulls/comments.md

```diff
@@ -62,10 +62,15 @@ Name | Type | Description
 Name | Type | Description
 -----|------|--------------
 `body`|`string` | **Required**. The text of the comment
-`commit_id`|`string` | **Required**. The SHA of the commit to comment on.
+`commit_id`|`string` | **Required**. The SHA of the latest commit of the pull request to comment on.
 `path`|`string` | **Required**. The relative path of the file to comment on.
 `position`|`integer` | **Required**. The line index in the diff to comment on.
 
+Note: When passing the commit_id, use the SHA of the latest commit in the pull request or your comment may appear as "outdated" if the specified position has been modified in a subsequent commit.
+
+Note: If you want to comment on a specific line, and you only have the line of the file and not the `position`. You can translate a file line to a `position` by parsing the `patch` returned from commits endpoint.
```

Comment:
Hi @tomasbjerre, what do you think about removing this note from this pull request and addressing it in a subsequent PR? I'd like to start a discussion with some of my colleagues to see if there is an easier way.

That way we can get this merged for your original comment about the `commit_id`. Does that sound good?